lib/commands/migrate.js

```diff
@@ -1,42 +1,81 @@
 "use strict";
 
 const fs = require("fs");
+const path = require("path");
 const chalk = require("chalk");
 const diff = require("diff");
 const inquirer = require("inquirer");
 const PLazy = require("p-lazy");
 const Listr = require("listr");
 
-const validate = require("webpack").validate;
-const WebpackOptionsValidationError = require("webpack")
-	.WebpackOptionsValidationError;
+const { validate } = require("webpack");
+const { WebpackOptionsValidationError } = require("webpack");
 
 const runPrettier = require("../utils/run-prettier");
 
 /**
-*
-* Runs migration on a given configuration using AST's and promises
-* to sequentially transform a configuration file.
-*
-* @param {String} currentConfigPath - Location of the configuration to be migrated
-* @param {String} outputConfigPath - Location to where the configuration should be written
-* @param {Object} options - Any additional options regarding code style of the written configuration
+ *
+ * Runs migration on a given configuration using AST's and promises
+ * to sequentially transform a configuration file.
+ *
+ * @param {Array} args - Migrate arguments such as input and
+ * output path
+ * @returns {Function} Runs the migration using the 'runMigrate'
+ * function.
+ */
 
-* @returns {Promise} Runs the migration using a promise that will throw any errors during each transform
-* or output if the user decides to abort the migration
-*/
+module.exports = function migrate(...args) {
+	const filePaths = args ? args.slice(3) : null;
+	if (!filePaths.length) {
+		throw new Error("Please specify a path to your webpack config");
```

Comment:
Instead of throwing an error which generates a long non-useful (in this case) stack trace, how about using chalk to show a one-line error message and exit the process gracefully? :wink:

Reply:
![image](https://user-images.githubusercontent.com/5961873/38176418-fa46256e-360b-11e8-8be0-09ac7bc0d198.png)

Reply:
Stack traces are good for us when people submit bugs. Console.errors when we're inside a promise

Reply:
This looks ugly though, error msg is intuitive enough for us to debug

Reply:
Yeah but in here we are intentionally showing an error to the user, not an error due to a bug in our code. I think we should separate the two concerns

Reply:
I'm convinced both ways. :smile_cat: